main.cc

```diff
@@ -776,7 +776,7 @@ update()
 
     auto *t = tools[pl.active_tool_slot];
     if (t) {
-        t->update(ship);
```

Comment:
I was doing this because the first iteration of inspection looked at the net. You need a ship to do so.

Reply:
This code here doesn't know what ship you want though -- you can only currently pretend it does because there's exactly one.